tools/ci_build/github/linux/ort_minimal/build_minimal_ort_android_baseline_and_report_bin_size.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# This script will run a baseline minimal ort build for android arm64-v8a ABI
+# and report the binary size to the ort mysql DB
+
+set -e
+
+# Create an empty file to be used with build --include_ops_by_config, which will include no operators at all
+echo -n > /home/onnxruntimedev/.test_data/include_no_operators.config
+
+# Run a baseline minimal build of ORT Android arm64-v8a
+# Generate binary size as /build/MinSizeRel/binary_size_data.txt
+python3 /onnxruntime_src/tools/ci_build/build.py \
+    --build_dir /build --cmake_generator Ninja \
+    --config MinSizeRel \
```

Comment:
Do we need to use Ninja on Linux? Probably doesn't matter but I thought we only had to resort to that on Windows.

Reply:
Seems build with Ninja is slightly faster and it is used everywhere in the linux CIs